### PR TITLE
Reproducible build blog

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
A blog on the work that has been done and why on Reproducible Builds at Eclipse Adoptium, and what future projects are planned.

### CODE REVIEW

**Comment by karianna** [comment]
@andrew-m-leonard Looks like a bunch of failing tests

File: content/blog/adoptium-reproducible-builds/index.md
Comment (smlambert) [inline content/blog/adoptium-reproducible-builds/index.md:34]: Do you want to state it explicitly that "if the builds from the multiple secure environments are identical to the one from the open-source projects, the likelihood of malicious tampering would be extremely low".

File: content/blog/adoptium-reproducible-builds/index.md
Comment (smlambert) [inline content/blog/adoptium-reproducible-builds/index.md:107]: May do well to have a short closing paragraph to:
- wrap-up/summarize the blog post 
- invite input on how consumers are securing their own supply chains 
- ask for feedback on the plans outlined in this post and how consumers will leverage this work

**Review by smlambert — COMMENTED** [review]
Reads very well @andrew-m-leonard, nice work!  I have added a few comments, and also wonder if a diagram might be beneficial to break up the text a bit (even if its a fairly generic one breaking down the 'product' into its sub-components... similar to the 'wheelbarrow' example from https://www.techtarget.com/searcherp/definition/bill-of-materials-BoM).

**Comment by andrew-m-leonard** [comment]
> @andrew-m-leonard Looks like a bunch of failing tests

@karianna yeah, 1st time doing one of these, I will trawl through all those...!

File: content/blog/adoptium-reproducible-builds/index.md
Comment (tellison) [inline content/blog/adoptium-reproducible-builds/index.md:42]:
```suggestion
cross checked with the SBOM for any given release to determine if they are affected.
```

File: content/blog/adoptium-reproducible-builds/index.md
Comment (tellison) [inline content/blog/adoptium-reproducible-builds/index.md:57]:
```suggestion
has not had any unforeseen effect.
```

File: content/blog/adoptium-reproducible-builds/index.md
Comment (tellison) [inline content/blog/adoptium-reproducible-builds/index.md:68]:
```suggestion
jdk17u stream, since jdk-17 is a long term supported release.
```

File: content/blog/adoptium-reproducible-builds/index.md
Comment (tellison) [inline content/blog/adoptium-reproducible-builds/index.md:89]:
```suggestion
integrate changes into the build scripts to fully support reproducible builds.
```

File: content/blog/adoptium-reproducible-builds/index.md
Comment (tellison) [inline content/blog/adoptium-reproducible-builds/index.md:91]: Maybe a link to an epic on the subject?

File: content/blog/adoptium-reproducible-builds/index.md
Comment (tellison) [inline content/blog/adoptium-reproducible-builds/index.md:95]: Consider adding links to issues, you may get people interested in contributing to some of these.

**Review by tellison — COMMENTED** [review]
A few comments, and some style suggestions.
Take as you wish :-)

File: content/blog/adoptium-reproducible-builds/index.md
Comment (sxa) [inline content/blog/adoptium-reproducible-builds/index.md:105]:
```suggestion
- Further exploring required fixes to non-deterministic build output on other platforms, e.g.Windows.
```
I'd be tempted to call out Windows explicitly as something we intend to persue for definite, and maybe mention others in a separate bullet point, as it would be particularly good to encourage anyone interested in e.g. AIX to step forward and help :-) (I might talk to Tyler explicitly about that one)

**Review by sxa — COMMENTED** [review]
LGTM generally - most of my comments are nitpicks so I won't block on them :-)

The only overall comment I would make is that it's a bit light on 'practical detail' of the difficulties you encountered and how someone could do this for themselves since it's quite high level. It's almost worth splitting this into two blog posts - one with the high level "Secure supply chain" topic and another which goes into more of the underlying details of the reproducible bit - compiler options, problems you encountered, debugging them etc., which could help others who want to apply similar techniquies to their own projects, and potentially a link to the video showing people how to use our scripts to produce reproducible versions of the builds. The AMD vs Intel ordering in particular would be a good one to call out somewhere.

Hopefully that covers all the things I said to you after my initial review outside github - whever that was!

File: content/blog/adoptium-reproducible-builds/index.md
Comment (sxa) [inline content/blog/adoptium-reproducible-builds/index.md:25]: One thing I forgot in my last review - perhaps include a list of the sort of things that are included in an SBOM for people who have never heard of it before (or an external reference if that's more applicable, and/or a link to one of ours :-) )

File: content/blog/adoptium-reproducible-builds/index.md
Comment (andrew-m-leonard) [reply content/blog/adoptium-reproducible-builds/index.md:4]: added

File: content/blog/adoptium-reproducible-builds/index.md
Comment (sxa) [inline content/blog/adoptium-reproducible-builds/index.md:111]:
```suggestion
Today's Enterprise Software needs to be more secure and safe from vulnerability attacks than ever before. Providing methods
```

**Review by tellison — APPROVED** [review]

**Comment by tellison** [comment]
@andrew-m-leonard I'll leave it to you to decide if this is finished and ready to merge/publish. I've had another read through now and given it a +1 for my pov.

**Comment by andrew-m-leonard** [comment]
@sxa @smlambert If you're happy with this, please can one of you merge it? as I don't have write access. thanks

**Comment by smlambert** [comment]
Thanks @andrew-m-leonard!  This is a fantastic start to an entire series of related posts from the SSDF topic area.

**Pull request merged**
